network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/space_card_listing_block.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="container tw-py-4 medium:tw-py-7">
+  <h3 class="tw-font-semibold tw-mb-4">{{ value.title }}</h3>
```

Comment:
Small suggestion/question: I'm not sure of the context of this component within the page, but if there's a chance this won't be preceded by an `<h2>` in the page heading hierarchy, I'd swap this for an `<h2>` to be safe.